README.md

```diff
@@ -118,6 +145,62 @@ TODO
 TODO
 -->
 
+## GridMaterial
+
+_extends THREE.MeshPhongMaterial_
+
+A material with the same interface as `THREE.MeshPhongMaterial`. It's used to add a grid on the mesh, created by the method `csgGridShaderMixin` passing a shader. Default is enabled but if you want you can disable the grid with
+```js
+gridMat.enableGrid = false;
+```
+
+## Helpers
+<!--
+## PointsHelper
+
+_extends THREE.InstancedMesh_
+
+Helper class 
+
+### .setPoints
+
+```js
+setPoints( points: Vector3 ) : void;
+```
+
+Sets the 
+-->
+
+## EdgesHelper
+
+_extends THREE.LineSegments_
+
+Helper class for generating a line to display the intersection edges. If no edges are provided then an empty array will be passed.
+
+### .setEdges
+
+```js
+setEdges( edges: EdgesGeometry[] ) : void
+```
+
+Sets the geometry of the helper with the points of the intersection of the edges. First gets the points of the edges, disposes of geometry and then sets the attributes for this geometry from an array of points of the edges.
```

Comment:
I think we can keep this simple and just say that this function sets the list of lines to be visualized.